lib/src/embedded_cli.c

```diff
@@ -194,6 +211,26 @@ static const uint16_t cliInternalBindingCount = 1;
 
 static const char *lineBreak = "\r\n";
 
+/* Reference for VT100 escape sequences: https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences */
```

Comment:
Please also link [ECMA-48](https://ecma-international.org/publications-and-standards/standards/ecma-48)

Reply:
Done